### ISSUE TITLE
Replicator panic

### ISSUE THREAD

**Issue by HM999** [solana-labs/solana#4949]
Followed the steps on the website exactly. It ran for about 15 minutes before:
```
thread 'main' panicked at 'transfer didn't work!: Custom { kind: Other, error: StringError("retry_transfer failed in 10 retries") }', src/libcore/result.rs:997:5
```
I am running again with RUST_BACKTRACE, will post results.

Platform is Ubuntu 18.02, used solana installer without arguments (i.e. latest), on AWS so no internet outage etc.

**Comment by HM999** [comment]
```
thread 'main' panicked at 'transfer didn't work!: Custom { kind: Other, error: StringError("retry_transfer failed in 10 retries") }', src/libcore/result.rs:997:5
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:39
   1: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:71
   2: std::panicking::default_hook::{{closure}}
             at src/libstd/sys_common/backtrace.rs:59
             at src/libstd/panicking.rs:197
   3: std::panicking::default_hook
             at src/libstd/panicking.rs:211
   4: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:474
   5: std::panicking::continue_panic_fmt
             at src/libstd/panicking.rs:381
   6: rust_begin_unwind
             at src/libstd/panicking.rs:308
   7: core::panicking::panic_fmt
             at src/libcore/panicking.rs:85
   8: core::result::unwrap_failed
   9: solana::replicator::Replicator::run
  10: solana_replicator::main
  11: std::rt::lang_start::{{closure}}
  12: std::panicking::try::do_call
             at src/libstd/rt.rs:49
             at src/libstd/panicking.rs:293
  13: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:87
  14: std::rt::lang_start_internal
             at src/libstd/panicking.rs:272
             at src/libstd/panic.rs:388
             at src/libstd/rt.rs:48
  15: main
  16: __libc_start_main
  17: _start
```

**Comment by mvines** [comment]
ref: https://github.com/solana-labs/solana/blob/fbea9d8621076e5a0d0f3ddf83a575d4b48bbdd4/core/src/replicator.rs#L535

@sagar-solana - can you please take a look.  We certainly shouldn't be crashing on a failed transaction.

**Comment by sagar-solana** [comment]
Yeah I'll silence this. Shouldn't be fatal. But generally indicates that the network is down/unresponsive. I'll fix it asap.

**Comment by sagar-solana** [comment]
Should be fixed with #4957
